### PR TITLE
Fix startup time-warp issue

### PR DESCRIPTION
**Pull request by penrods**
Raspberry Pi's don't have a built-in clock, so at boot-up the clock just picks up from when they were last running.  Normally this is corrected very quickly by NTP from an internet server, but if there is no network connection that cannot happen.

When an out-of-the-box Mark 1 or Picroft is being setup, the clock is set to whenever the image was created.  Upon completing the Wifi setup step the NTP service can finally sync with the internet, so time suddenly "jumps" to weeks later -- usually.  In either case (when the date jumps or when the date is erronously months old), there is potential for havoc.

These changes deal with that situation.  Upon network connection, an NTP synchonization is forced.  If it is detected that a major time jump happened
(more than 1 hour), then the user is notified that the clock change requires
a reboot and the system restarts.

Other changes:
* use the new "system." message namespace
* add pause before the system.reboot during a WIPE, allowing reset to totally complete

### CODE REVIEW

File: mycroft/skills/main.py [hunk 1]
```diff
@@ -97,6 +103,36 @@ def check_connection():
         Runs as a Timer every second until connection is detected.
     """
     if connected():
+        enclosure = EnclosureAPI(ws)
+
+        # Force a sync of the local clock with the internet
+        enclosure.mouth_text(mycroft.dialog.get("message_synching.clock"))
```
[the remark this answers is not in the document]
Comment (penrods) [reply mycroft/skills/main.py]: I'm not following this comment.  What do you mean?

[the remark this answers is not in the document]
Comment (penrods) [reply mycroft/skills/main.py]: FYI, I fixed the text in the help message in mycroft/messagebus/send.py (I assume that is what you meant in the top comment).  Good catch.

[the remark this answers is not in the document]
Comment (MatthewScholefield) [reply mycroft/skills/main.py]: The filenames match fine. I'm just talking about the spelling: sync*h*ing

[the remark this answers is not in the document]
Comment (penrods) [reply mycroft/skills/main.py]: Ahh.  I'm sticking up for the synch'ers of the world!  It is a legit spelling, especially for us Gen-X programmers.  :)

https://english.stackexchange.com/questions/681/synced-or-synched

[the remark this answers is not in the document]
Comment (MatthewScholefield) [reply mycroft/skills/main.py]: Ah, ok. The limit to my research was typing it in on Google xD.
image

**Pull request merged**